Workbooks/WorkbooksMetadata.json

```diff
@@ -2204,5 +2204,18 @@
                 "Platform"
             ]
         }
-    }      
+    },   
+    {
+      "workbookKey": "MicrosoftSentinelCostEUR",
+      "logoFileName": "Azure_Sentinel.svg",
+      "description": "This workbook provides an estimated cost in EUR (€) across the main billed items in Microsoft Sentinel: ingestion, retention and automation. It also provides insight about the possible impact of the Microsoft 365 E5 offer.",
+      "dataTypesDependencies": [],
+      "dataConnectorsDependencies": [],
+      "previewImagesFileNames": [ "MicrosoftSentinelCostEURWhite.png", "MicrosoftSentinelCostEURBlack.png"],
+      "version": "1.6.0",
```

Comment:
@MarcoPassanisi, can you please check the version, it should be 1.0.0 as this is been added for first time, thanks.

Reply:
Fixed